index.js

```diff
@@ -0,0 +1,71 @@
+const https = require('https');
+
+const token = process.env.CIRCLECI_TOKEN;
+const target_repo = process.env.TARGET_REPO;
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
  Identifier 'target_repo' is not in camel case                     camelcase